common/handler_local.py

```diff
@@ -20,7 +20,7 @@ def get_best_score_imgs(self, results: list[ScoredPoint]) -> list[Image.Image]:
         Handler for returning images with the highest similarity scores from local storage.
         Additionally, filenames are returned as future captions in front-end module.
         """
-        object_list = [r.payload["file"].replace('\\', '/') for r in results]
+        object_list = [r.payload["file"].replace("\\", "/") for r in results]
```

Comment:
ditto

Reply:
`r.payload` comes from Qdrant and if you generate entries on Windows, current indexing will pull the Windows path with `\` as a folder separator. This should be left as a precaution when working locally on Windows.